client/src/Components/HomePage/HomePage.js

```diff
@@ -19,26 +19,21 @@ class HomePage extends React.Component{
 	}
 	render(){
 		let ele = null
-		if (!this.state.enter){
-			ele = <div class='entry-main'>
+		
+			return <div className='entry-main'>
+				
 			<div className='entry-container'>
 				<img src={mainImage} />
 				<div className='entry-name'>
 					<p className='entry-title'>10x Team 7</p>
+					<Link to='/postList'>
 					<button onClick={this.toggleDisplay}>Enter</button>
```

Comment:
Is onclick required now?

Reply:
sir actually there is some state management required so i need to pass on a function which i couldnt figure out without this approach i was reading and answers which were suggested were using redux so that is why i did this for simple routng i have used react router